libpod/runtime.go

```diff
@@ -394,3 +549,15 @@ func (r *Runtime) Info() ([]InfoData, error) {
 
 	return info, nil
 }
+
+// SaveDefaultConfig saves a copy of the default config at the given path
+func SaveDefaultConfig(path string) error {
```

Comment:
What do you plan to do with this?

Reply:
Mostly there just in case we ever change the config and need to regenerate a default one. It seems useful to keep around

Reply:
Ok